### ISSUE TITLE
Add little site icons to the Web UI

### ISSUE THREAD

**Issue by M-Ashrafpour** [AdguardTeam/AdGuardHome#5187]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### Description

Hi, It would be great if you could add site icons ( favicon ) to the main page, query logs
and other pages, just like Adguard dns web dashboard ( "Requests  types" section and "Top companies" ).
It's a little thing, but it makes the UI a lot better and more enjoyable to look at.

[Example](https://camo.githubusercontent.com/b2c5b7b480e48352f206dd804c5a2bf34334c534859c58bdd4f69875f2d75c0f/68747470733a2f2f63646e2e616467756172642e636f6d2f636f6e74656e742f626c6f672f61727469636c65732f73746174735f656e2e706e67)

**Comment by agneevX** [comment]
Do you mean tab icons? Because that's not what favicons mean.

**Comment by M-Ashrafpour** [comment]
> Do you mean tab icons? Because that's not what favicons mean.

Yes. Yes. Are you sure? because that is what the Oxford dictionary and [Wikipedia](https://upload.wikimedia.org/wikipedia/commons/1/1d/Wikipedia_favicon_in_Firefox_on_KDE.png) say.

**Comment by lutfor-diu** [comment]
i think he means these icons attached in the image. nice to have if possible.
![Uploading Screenshot_1.png…]()

**Comment by lutfor-diu** [comment]
i think he means these icons attached in the image. nice to have if possible.
![Screenshot_1](https://user-images.githubusercontent.com/32225836/204120727-24742a7e-2e3f-4e32-b320-ccbc8a3923d6.png)

**Comment by M-Ashrafpour** [comment]
> i think he means these icons attached in the image. nice to have if possible. ![Screenshot_1](https://user-images.githubusercontent.com/32225836/204120727-24742a7e-2e3f-4e32-b320-ccbc8a3923d6.png)

Thank you. This is exactly what i mean.

**Comment by candrapersada** [comment]
and add icon AdGuardHome on `AdGuardHome.exe`

**Comment by lutfor-diu** [comment]
> > i think he means these icons attached in the image. nice to have if possible. ![Screenshot_1](https://user-images.githubusercontent.com/32225836/204120727-24742a7e-2e3f-4e32-b320-ccbc8a3923d6.png)
> 
> This is exactly what I meant.

any update regarding this request ?

**Comment by M-Ashrafpour** [comment]
> > > i think he means these icons attached in the image. nice to have if possible. ![Screenshot_1](https://user-images.githubusercontent.com/32225836/204120727-24742a7e-2e3f-4e32-b320-ccbc8a3923d6.png)
> > 
> > 
> > This is exactly what I meant.
> 
> any update regarding this request ?

unfortunately no
